api/swagger/swagger.yaml

```diff
@@ -1388,3 +1406,32 @@ definitions:
         type: array
         items:
           $ref: "#/definitions/Board"
+  BoardReport:
+    type: object
+    required:
+      - reportedBoardData
+      - whistleblowerData
+      - reportReason
+    properties:
+      reportedBoardData:
+        type: object
+        properties:
+          boardName:
+              type: string
+          boardAuthor:
+              type: string
+          boardDescription:
+              type: string
+          boardId:
+              type: string
+          boardUrl:
+              type: string
+      whistleblowerData:
+        type: object
+        properties:
+          whistleblowerName:
+              type: string
+          whistleblowerEmail:
+              type: string
+      reportReason:
+        type: string
```

Comment:
this data structure should be all renamed. Since you are `post` for a `board` it is redundant to include the word `board` for every property. Your data structure should be as simple as possible, like: 
```
{
  "id": "string",
  "name": "string",
  "author": "string",
  "url": "string",
  "description": "string",
  "reason": "string",
  "whistleblower": {
      "name": "string",
      "email": "string"
    }
}
```